shimmingtoolbox/dicom_to_nifti.py

```diff
@@ -0,0 +1,20 @@
+# coding: utf-8
+
+# TODO: check in unit test if dcm2bids_scaffold is installed, and also check for the required version.
+
+import subprocess
+
+from shimmingtoolbox import __dir_config_dcm2bids__
+
+
+def dicom_to_nifti(path_dicom, path_nifti, subject_id='sub-01', path_config_dcm2bids=__dir_config_dcm2bids__):
+    """ Converts dicom files into nifti files by calling dcm2bids
+
+    Args:
+        path_dicom (str): path to the input dicom folder
+        path_nifti (str): path to the output nifti folder
+
+    """
+    cmd = 'dcm2bids -d {} -p {} -o {} -l DEBUG -c {}'.format(
+        path_dicom, subject_id, path_nifti, path_config_dcm2bids)
```

Comment:
The config file is really a sort of miniature sorting program, tailored to the dataset. Hard-coding it doesn't make sense to me. It seems like maybe we should be generating each time based on details we scan from the dataset?

Reply:
it used to be generated each time, then i suggested to remove that automatic generation, because depending on the MR sequence version, some fields don't follow the same convention. so i'm not 100% sure we can auto-generate that. 

at this point, i would say: let's not spend time on this config file. We have one hard-coded now, and it works. In the future we could re-think the auto-generation after we are back to the scanner and can test the various pulse sequences